packages/cli/src/constructs/playwright-check.ts

```diff
@@ -76,6 +77,7 @@ export interface PlaywrightCheckProps extends RuntimeCheckProps {
    *
    * @example "E2E Tests"
    * @example "Critical User Flows"
+   * @deprecated Use groupId instead for better consistency
```

Comment:
This is not what we want. `groupId` is a deprecated property. What we want to support is the `group` property. Fixing this will also have some implications on the rest of the implementation.

Also, please try to use `{@link}` when making references. Like this:

```suggestion
   * @deprecated Use {@link group} instead.
```